### DISCUSSION TITLE
R package "magick" not available

### DISCUSSION THREAD

**Discussion by ciaranmci** [opensafely/documentation#610] [General]
My analysis script makes use of an [R package called magick](https://cran.r-project.org/web/packages/magick/vignettes/intro.html). When I run my files locally, an error is thrown saying <there is no package called 'magick'>.
I assume magick hasn't been installed.
**Please, can the magick package be installed?**

**Comment by StevenMaude** [comment]
@ciaranmci: :wave: I don't manage the R Docker image, but have opened an [issue in its repository](https://github.com/opensafely-core/r-docker/issues/67) on your behalf.
